### ISSUE TITLE
App: Visemes not working for some models

### ISSUE THREAD

**Issue by madjin** [webaverse/app#946]
Testing a VRM which didn't have working visemes in Webaverse but worked in other VRM programs such as vmagicmirror

![VMagicMirror_7kC8lYqnlT](https://user-images.githubusercontent.com/32600939/110004142-f412a500-7ccb-11eb-9319-8ba3a6fdc48c.png)

Here's a side by side look in blender with shapekeys menu open between a VRM with working visemes and the model that doesn't have working visemes in Webaverse:

![picrf2jSYI](https://user-images.githubusercontent.com/32600939/110004227-0b519280-7ccc-11eb-9dc6-d39187285435.png)

Moving and/or renaming them doesn't seem to have an effect either - checked in chrome / firefox and with diff audio devices as well just in case. Tested a VRM with the same blendshape names as the working Miku model and it didn't work, so don't think its related to the names.

**Comment by madjin** [comment]
Maybe it's looking for shapekeys on a certain mesh name, working VRMs have shapekeys on Face.baked while the one without working visemes only has body.baked.

**Comment by avaer** [comment]
Everything is name based. there are like 10 issues on this now :P.

Someone needs to go in and make the mapping not name-based, but pulled from the VRM.

**Comment by madjin** [comment]
https://github.com/webaverse/app/commit/50444de5b32d459667cf48724c3801b3fada9cbc most recent commit related to visemes

Okay, Via suggested that for visemes it will be more ideal to be pulling from the VRM blendshape proxy script because then we can route expressions other than visemes through it, and it'll be much much more compatible with models.

**Comment by avaer** [comment]
Yes I agree with Via that would be better. It just needs to be done.

**Comment by avaer** [comment]
Basically this should be pulled from the VRM data: https://github.com/webaverse/app/blob/cf7b7310d671fcefb37ad13a97eea99ca55c823b/avatars/avatars.js#L1535
